projects.go

```diff
@@ -472,6 +472,7 @@ type CreateProjectOptions struct {
 	UseCustomTemplate                         *bool             `url:"use_custom_template,omitempty" json:"use_custom_template,omitempty"`
 	GroupWithProjectTemplatesID               *int              `url:"group_with_project_templates_id,omitempty" json:"group_with_project_templates_id,omitempty"`
 	BuildCoverageRegex                        *string           `url:"build_coverage_regex,omitempty" json:"build_coverage_regex,omitempty"`
+	RemoveSourceBranchAfterMerge              *bool             `url:"remove_source_branch_after_merge,omitempty" json:"remove_source_branch_after_merge,omitempty"`
```

Comment:
Could you reorder these a little to keep the ordering inline with the docs? That would mean moving the `BuildCoverageRegex` field above the `CIConfigPath` and the new `RemoveSourceBranchAfterMerge` field that you just added above `LFSEnabled`. Would be great if we can do that before we merge it, thanks!

Reply:
I will do it next week.